docs/manual/common/guide/build/UpgradingAkkaPlay.md

```diff
@@ -0,0 +1,211 @@
+# Using newer versions of Akka and Play Framework
+
+Lagom relies on Akka and Play Framework for some of its implementation.
+
+The versions of Akka and Play that ship with Lagom should usually be sufficient to run a Lagom application. However, in some cases it can be useful to upgrade them while staying on the same version of Lagom. This is usually possible when Akka and Play maintains binary compatibility between releases.
```

Comment:
In `lightbend-markdown` (unlike paradox or general markdown) a single-newline becomes a `<br>` and a double-newline becomes a new `<P>` so sentences are better of on a single source line.